interfaces/Portalicious/src/app/pages/project-registration-activity-log/components/activity-log-expanded-row/activity-log-expanded-row.component.ts

```diff
@@ -113,7 +116,9 @@ export class ActivityLogExpandedRowComponent
           },
           {
             label: $localize`FSP`,
-            value: item.attributes.fsp,
+            value: this.translatableStringService.translate(
+              item.attributes.financialServiceProviderConfigurationLabel,
+            ),
```

Comment:
```suggestion
            value: item.attributes.financialServiceProviderConfigurationLabel,
```

IIRC you don't need to translate. If you pass in a translatable string, the underlying component will try to translate it. (Needs to be tested).